enterprise/management/commands/migrate_enterprise_catalogs.py

```diff
@@ -60,6 +60,7 @@ def handle(self, *args, **options):
                 client.create_enterprise_catalog(
                     str(enterprise_catalog.uuid),
                     str(enterprise_catalog.enterprise_customer.uuid),
+                    enterprise_catalog.enterprise_customer.name,
```

Comment:
I think we'll need to update this management command to update enterprise catalogs as well. That way we can make sure that they all have an enterprise customer name attached to them

Reply:
True. The management command, as is, only creates new catalogs but doesn't update existing ones. Will work that into this PR as well.

Reply:
I updated the management command to update with a PUT request if a corresponding catalog already exists in the service. If the related catalog does not yet exist, it will be created.